### PR TITLE
Update: Botbreak / Was broken on rocknix and maybe some other cfws

### PR DESCRIPTION
**Pull request by Damian2809**
# New Update for Botbreak

## Game Information
- **Title**: Botbreak

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [ ] AmberELEC
- [ ] ArkOS
- [X] ROCKNIX
- [ ] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [ ] 640x480
- [ ] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/botbreak/Botbreak.sh [hunk 1]
```diff
@@ -14,38 +14,27 @@ else
...
-export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$GAMEDIR/utils/libs:$LD_LIBRARY_PATH"
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$LD_LIBRARY_PATH"
 export GMLOADER_DEPTH_DISABLE=1
 export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
 
 cd $GAMEDIR
-
-# Check for file existence before trying to manipulate them:
-[ -f "./gamedata/data.win" ] && mv gamedata/data.win gamedata/game.droid
-[ -f "./gamedata/game.win" ] && mv gamedata/game.win gamedata/game.droid
```
Comment (JeodC) [inline ports/botbreak/Botbreak.sh:37]: This is still necessary isn't it? It would be data.win or game.unx and must be named game.droid.

**Review by JeodC — CHANGES_REQUESTED** [review]

File: ports/botbreak/Botbreak.sh (hunk 1 above)
Comment (Damian2809) [reply ports/botbreak/Botbreak.sh:37]: The Game is ready to run
so its already named game.droid

Comment (JeodC) [reply ports/botbreak/Botbreak.sh:37]: Oh I was thinking of botvice. Carry on.

File: ports/botbreak/Botbreak.sh [hunk 2]
```diff
@@ -14,38 +14,27 @@ else
...
-
-# Make sure uinput is accessible so we can make use of the gptokeyb controls
-$ESUDO chmod 666 /dev/uinput
-
-$GPTOKEYB "gmloader" -c ./botbreak.gptk &
-
 $ESUDO chmod +x "$GAMEDIR/gmloader"
 
+$GPTOKEYB "gmloader" -c ./botbreak.gptk &
+pm_platform_helper "gmloader"
```
Comment (kloptops) [inline ports/botbreak/Botbreak.sh:37]: `pm_platform_helper` needs an absolute path:
```suggestion
pm_platform_helper "$GAMEDIR/gmloader"
```

**Review by kloptops — CHANGES_REQUESTED** [review]

**Review by Cebion — APPROVED** [review]

**Review by JeodC — APPROVED** [review]

**Pull request merged**